service.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>reside-eng/renovate-config",
+    "packages:react",
+    "group:graphql-toolsMonorepo",
+    "group:material-uiMonorepo",
+    "group:storybookMonorepo",
+    "group:nextjsMonorepo",
+    "group:emotionMonorepo"
+  ],
+  "packageRules": [
+    {
+      "description": "Automerge non-major npm dev dependencies off business hours",
+      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+      "matchDatasources": ["npm"],
+      "matchDepTypes": ["devDependencies"],
+      "schedule": [
+        "after 10pm and before 5am every weekday",
+        "every weekend"
+      ],
+      "automerge": true,
+      "automergeType": "pr",
+      "automergeStrategy": "squash"
+    },
+    {
+      "description": "Automerge patch npm dependencies off business hours on weekday mornings",
+      "matchUpdateTypes": ["patch", "pin", "digest"],
+      "matchDatasources": ["npm"],
+      "matchDepTypes": ["dependencies"],
+      "schedule": [
+        "after 6am and before 8am every weekday"
+      ],
+      "automerge": true,
+      "automergeType": "pr",
+      "automergeStrategy": "squash"
+    },
+    {
+      "description": "Automerge non-major Github Actions dependencies off business hours",
+      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+      "matchDepTypes": ["action"],
+      "schedule": [
```

Comment:
Any particular reason this schedule needs to be different from the one above?

Reply:
Dependencies are only on weekday mornings so that engineers will be available to address bugs if they come up - since this will be used in the repos with CD. The actions dependencies are set to non business hours including the weekend (for these we just don't want to cause the need to update branch and rebuild for engineers PRs multiple times during the day)